roles/bootstrap-os/tasks/bootstrap-coreos.yml

```diff
@@ -3,46 +3,50 @@
   raw: stat /opt/bin/.bootstrapped
```

Comment:
files/bootstrap.sh adds it. i didn't change anything with the coreos playbooks. i just combined them into a bootstrap-os role (if i'm understanding your question correctly).